docs/examples/container-config.json

```
@@ -3,7 +3,7 @@
     "name": "busybox"
   },
   "image":{
-    "image": "busybox"
+    "image": "index.docker.io/busybox"
```

The `index` subdomain is not required from my point of view: 
```suggestion
    "image": "docker.io/busybox"
```

---

@vorburger could you remove the `index` prefix?

---

@feiskyer done. (Sorry it took a moment.) Merge?

---

It looks like it's still there :)